x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_forms/config_form.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiHorizontalRule,
+  EuiSpacer,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import styled from 'styled-components';
+
+const PolicyDetailCard = styled.div`
+  .policyDetailTitleOS {
+    flex-grow: 2;
```

Comment:
Eui Flex component support the `grow` prop, which I think does what this is attempting to do.